### ISSUE TITLE
About dataset

### ISSUE THREAD

**Issue by LiuZR10** [renjiaoyi/imagerelighting#4]
首先，感谢您开源您这篇优秀文章的代码。在这里我有几个问题要问您：首先就是您的这个模型是否具有泛化性；其次，您的工作最后对比实验的数据集图片是什么数据集。希望您能够解答我的疑惑，谢谢您

**Comment by LiuZR10** [comment]
另外，是不是我将invrender.pth放入path中，就可以直接用test_end2end_invdender.py在数据集上测试，不用再训练

**Comment by renjiaoyi** [comment]
> 另外，是不是我将invrender.pth放入path中，就可以直接用test_end2end_invdender.py在数据集上测试，不用再训练

是的。可以泛化，不需重新训练或者微调模型。最后一个图（图6）中用来进行relighting evaluation的数据是渲染的虚拟物体。

**Comment by LiuZR10** [comment]
图五用到的是什么数据集呢

**Comment by LiuZR10** [comment]
咱们的训练集是所有Relit dataset里的图片吗，测试集是什么呢

**Comment by renjiaoyi** [comment]
> 图五用到的是什么数据集呢

是[30]这篇的数据

**Comment by renjiaoyi** [comment]
> 咱们的训练集是所有Relit dataset里的图片吗，测试集是什么呢

训练的时候是自监督，所有数据用于训练，没有单独分出测试集
